package.json

```diff
@@ -28,6 +28,7 @@
         "deploy": "hardhat deploy --network",
         "lint": "npm run lint:sol && npm run lint:ts",
         "lint:sol": "solhint 'contracts/**/*.sol'",
+        "lint:sol:fix": "solhint --fix 'contracts/**/*.sol'",
         "lint:sol:prettier": "prettier 'contracts/**/*.sol' --list-different",
         "lint:ts": "eslint 'src/**/*.ts' 'test/**/*.ts' --fix",
```

Comment:
Unrelated to this PR, but `--fix` in the TypeScript linting could hide issues in CI (since the linting step will pass despite the files in the commit requiring changes).

Can we have a `lint:fix = lint:sol:fix & lint:ts:fix` and only add the `--fix` parameter in a `lint:ts:fix` script?

Reply:
done